cmd/celestia-appd/cmd/modify_root_command_multiplexer.go

```diff
@@ -26,18 +33,22 @@ func modifyRootCommand(rootCommand *cobra.Command) {
 		panic(err)
 	}
 
+	var extraArgs []string
+	if v2UpgradeHeight > "0" {
```

Comment:
doing a `>` here looks like a foot gun, should this just be `if v2UpgradeHeight != ""` and default it to `""`?

Reply:
it would have lexicographical worked but I do agree that it looks like one. i can change yeah